socorro/unittest/signature/test_signature_utilities.py

```diff
@@ -1018,23 +968,8 @@ def test_create_frame_list_1(self):
         assert 'normalized' in frames_from_json_dump['frames'][0]
         assert frames_from_json_dump['frames'][0]['normalized'] == expected[0]
 
-    def test_create_frame_list_2(self):
-        config = self.get_config()
-        config.c_signature.maximum_frames_to_consider = 3
-        sgr = SignatureGenerationRule(config)
-        frame_signatures_list = sgr._create_frame_list(frames_from_json_dump)
-        expected = [
-            u'NtWaitForMultipleObjects',
-            u'WaitForMultipleObjectsEx',
-            u'WaitForMultipleObjectsExImplementation',
-        ]
-        assert frame_signatures_list == expected
-        assert 'normalized' in frames_from_json_dump['frames'][0]
-        assert frames_from_json_dump['frames'][0]['normalized'] == expected[0]
```

Comment:
This tests a configuration variable that we nixed, so we don't need the test anymore.